### ISSUE TITLE
2D graph pretraining: How to name wandb logs?

### ISSUE THREAD

**Issue by sid-betalol** [LeMaterial/lematerial-forgebench#83]
The title is self-explanatory; need to define a nomenclature that is easy to understand and specific enough.

My first pass would be {architecture}_{augmentation}_{loss}_{hyperparams}, but we can iterate.
